app/services/proxy_status/tftp.rb

```diff
@@ -0,0 +1,21 @@
+module ProxyStatus
+  class TFTP < Base
+    def initialize(proxy, opts = {})
+      super(proxy, opts, 'TFTP')
+    end
+
+    def tftp_server
+      raise ::Foreman::Exception.new(N_('No TFTP feature for %s'), proxy.to_label) unless proxy.has_feature?('TFTP')
```

Comment:
With latest change, this is a precaution of a precaution (e.g., if no tftp feature no chance it will get here...)